### PR TITLE
Remove total bytes from page source and record cursor

### PR DESCRIPTION
**Pull request by electrum**
The method is not used by anything in the engine.

### CODE REVIEW

File: presto-hive/src/main/java/com/facebook/presto/hive/orc/OrcPageSource.java [hunk 1]
```diff
@@ -110,12 +110,6 @@ public OrcPageSource(
     }
 
     @Override
-    public long getTotalBytes()
-    {
-        return recordReader.getSplitLength();
```
Comment (nezihyigitbasi) [inline presto-hive/src/main/java/com/facebook/presto/hive/orc/OrcPageSource.java]: After this change `OrcRecordReader::getSplitLength()` is only used in tests, we can remove that too.

File: presto-hive/src/main/java/com/facebook/presto/hive/rcfile/RcFilePageSource.java [hunk 2]
```diff
@@ -105,12 +105,6 @@ public RcFilePageSource(
     }
 
     @Override
-    public long getTotalBytes()
-    {
-        return rcFileReader.getLength();
```
Comment (nezihyigitbasi) [inline presto-hive/src/main/java/com/facebook/presto/hive/rcfile/RcFilePageSource.java]: With this change `RcFileReader::getLength()` can be removed too as it is not used by anywhere else.

File: presto-hive/src/main/java/com/facebook/presto/hive/orc/OrcPageSource.java (hunk 1 above)
Comment (electrum) [reply presto-hive/src/main/java/com/facebook/presto/hive/orc/OrcPageSource.java]: It's used by `OrcPageSource.getTotalBytes()`

Comment (electrum) [reply presto-hive/src/main/java/com/facebook/presto/hive/orc/OrcPageSource.java]: Nice catch on these. I'll check with @dain on if he wants to remove them.

**Pull request merged**